joplin/js/editor.js

```diff
@@ -202,9 +205,9 @@ $(function() {
         var langString = labelText.match(languageRegex);
 
         if (langString != null && langString != lowerLanguageStrings[currentLang]) {
-          elem.classList.add("hidden");
+          elem.parentElement.classList.add("hidden");
```

Comment:
ooh! is there an issue for the thing this fixes somewhere? do we get bonus magic?

Reply:
Oh yeah bonus magic. I don't think it was even an issue yet